cogs/greetings.py

```diff
@@ -20,3 +20,14 @@ async def man(self, ctx: Context):
     async def l(self, ctx: Context):
         # Sends a message to the channel using the Context object.
         await ctx.channel.send("L")
+
+    @commands.command()
+    async def choose(self, ctx: Context, *args):
+        # Sends a message to the channel using the Context object.
```

Comment:
Comments that describe what a function does is written as a string with tripple quotes, not #

"""Description here"""